release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,389 @@
+|===
+|Feature |{product-title-short} 4.5 |{product-title-short} 4.6 |{product-title-short} 4.7
+
+|API token authentication for {cloud-redhat-com}^[1]^
+|GA
+|DEP
+|DEP
+
+|`definitions.stackrox.io`
+|DEP
+|DEP
+|DEP
+
+|Google Container Registry integration^[2]^
+|GA
+|DEP
+|DEP
+
+|Kernel support packages and driver download functionality ^[3]^
+|DEP
+|DEP
+|DEP
+
+|Reporting of Istio vulnerabilities
+|DEP
+|DEP
+|DEP
+
+|`rhacs-collector-slim*` images^[4]^
+|DEP
+|DEP
+|REM
+
+|StackRox Scanner
+|GA
+|DEP
+|DEP
+
+|`/v1/clustercves/suppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/clustercves/unsuppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/suppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/nodecves/unsuppress` APIs^[5,6]^
+|DEP
+|DEP
+|DEP
+
+|`/v1/summary/counts` endpoint
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Management (1.0) menu item^[7]^
+|DEP
+|DEP
+|DEP
+
+|Vulnerability Report Creator permission
+|DEP
+|DEP
+|DEP
+
+|Scanner V4 support for openSUSE Leap 15.0 and 15.1
+|GA
+|DEP
+|REM
+
+|===
+
+[.small]
+--
+1. API token authentication is deprecated. The corresponding cloud source integration now uses service accounts for authentication.
+
+2. The Google Container Registry integration is deprecated in response to the deprecation of Container Registry. You can use the Artifact Registry as a registry replacement and Scanner V4 as a scanner replacement.
++
+For more information, see link:https://cloud.google.com/artifact-registry/docs/transition/transition-from-gcr[Transition from Container Registry] (Google Cloud documentation).
+
+3. Kernel support packages and driver download functionality are deprecated.
+
+4. The `rhacs-collector-slim*` image is deprecated and has been removed in {product-title-short} 4.7.0. `rhacs-collector*` image used to contain kernel modules and eBPF probes, but {product-title-short} no longer needs those items. 
+The `rhacs-collector*` and the `rhacs-collector-slim*` images are now functionally the same. 
+
+5. A feature flag controls this API object, and you can enable or disable this API object by using the `ROX_VULN_MGMT_LEGACY_SNOOZE` environment variable.
+
+6. The format for specifying duration in JSON requests to `v1/nodecves/suppress`, `v1/clustercves/suppress`, and `v1/imagecves/suppress` has been changed to the ProtoJSON format. 
+Only a numeric value representing seconds with optional fractional seconds for nanosecond precision and followed by the `s` suffix is supported. 
++
+For example, `0.300s`, `-5400s`, or `9900s`. The previously valid time units of `ns`, `us`, `µs`, `ms`, `m`, and `h` are no longer supported.
+
+7. The *Vulnerability Management* -> *Dashboard* view is deprecated and is planned to be removed in a future release. You can use the *Workload CVEs*, *Exception Management*, *Platform CVEs*, and *Node CVEs* views as alternatives.
+--
+
+[id="removed-features_{context}"]
+=== Removed features
+
+The following section provides information about the removed features:
```

Comment:
The format for explanations is now a footnote in the table and a numbered item below the table, to try to make correlating the table and the additional information easier to locate, so this would be # 8, and then add the 8 footnote to the item in the table?